src/index.html

```diff
@@ -7,8 +7,72 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet" 
+    crossorigin="true">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="conteiner" id="home">
+      <li class="nav__img">
+        <a  href="#home">
+          <img
+            class="img"
+            src="images/logo.png"
+            alt="Logo MOYO"
+          >
+        </a>
+      </li>
+      <nav class="navigation">
+        <ul class="navigaion__list">
```

Comment:
Follow BEM-methodology rules for naming classes
```suggestion
      <nav class="nav">
        <ul class="nav__list">
```